src/app/zone-ingresses/views/item/ServicesView.vue

```diff
@@ -15,102 +15,95 @@
 
       <KCard>
         <template #body>
-          <template
-            v-for="items in [
-              ...(props.data?.zoneIngress?.availableServices ? [props.data.zoneIngress.availableServices] : [])
+          <AppCollection
+            :empty-state-message="t('common.emptyState.message', { type: 'Services' })"
+            :headers="[
+              { label: 'Name', key: 'name' },
+              { label: 'Mesh', key: 'mesh' },
+              { label: 'Protocol', key: 'protocol' },
+              { label: 'No. Instances', key: 'instances' },
+              { label: 'Actions', key: 'actions', hideLabel: true },
             ]"
-            :key="items"
+            :items="props.data.zoneIngress.availableServices"
           >
-            <AppCollection
-              :empty-state-message="t('common.emptyState.message', { type: 'Services' })"
-              :headers="[
-                { label: 'Name', key: 'name' },
-                { label: 'Mesh', key: 'mesh' },
-                { label: 'Protocol', key: 'protocol' },
-                { label: 'No. Instances', key: 'instances' },
-                { label: 'Actions', key: 'actions', hideLabel: true },
-              ]"
-              :items="items"
-            >
-              <template #name="{ row: item }: {row: AvailableService}">
-                <template
-                  v-for="service in [item.tags?.['kuma.io/service'] ?? t('common.collection.none')]"
-                  :key="service"
-                >
-                  <RouterLink
-                    v-if="service.length > 0"
-                    :to="{
-                      name: 'service-detail-view',
-                      params: {
-                        mesh: item.mesh,
-                        service: service,
-                      },
-                    }"
-                  >
-                    {{ service }}
-                  </RouterLink>
-                  <template
-                    v-else
-                  >
-                    {{ service }}
-                  </template>
-                </template>
-              </template>
-
-              <template #mesh="{ row: item }: {row: AvailableService}">
+            <template #name="{ row: item }: {row: AvailableService}">
+              <template
+                v-for="service in [item.tags?.['kuma.io/service'] ?? t('common.collection.none')]"
+                :key="service"
+              >
                 <RouterLink
+                  v-if="service.length > 0"
```

Comment:
**Issue**: This can never be `false` unless the service annotation is actually the empty string. If the purpose of this is to be `true` if the service annotation is there, then line 31 above should use `?? null` instead of a `t(...)` result which can’t be predictable.

Reply:
Pretty sure these conditionals would never happen anyway, guessing that in order for one of these to be an `AvailableService` it would need to had a tag `kuma.io/service`. I'm just checking in with backend now, but I removed all this and altered the type to try to reflect this a little better. I'll wait for confirmation before merging this tho.

Oh I also changed `common.none` to be `''` instead of `' '`

Reply:
Just had confirmation that assuming that `kuma.io/service` is always set on `AvailableService` is ok